src/index.js

```diff
@@ -0,0 +1,109 @@
+import './styles.css';
+import createModal from './scripts/popup.js';
+import countMovies from './scripts/countMovies.js';
+
+const tvContainer = document.getElementById('tvContainer');
+const tvArray = [5, 169, 1871, 73, 51, 109, 156, 576];
+
+let responseArray = [];
+
+const readLikes = async () => {
+  const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Ql2WzJr90DiP5KlSpxzA/likes/', {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  const likes = await response.json();
+  responseArray = likes;
+};
+
+const newLike = async (itemnumber) => {
+  const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Ql2WzJr90DiP5KlSpxzA/likes/', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: `tv${itemnumber}`,
+    }),
+  });
+  return response; // Added this to avoid linter error
+};
+
+const logTV = async (item, index) => {
+  const response = await fetch(`https://api.tvmaze.com/shows/${item}`);
+  const movies = await response.json();
+  const card = document.createElement('section');
+  const image = document.createElement('img');
+  const midDiv = document.createElement('section');
+  const likeButton = document.createElement('section');
+  const title = document.createElement('p');
+  const likesCounter = document.createElement('p');
+  const buttonComments = document.createElement('button');
+  const buttonReservations = document.createElement('button');
+
+  buttonComments.innerHTML = 'Comments';
+  buttonReservations.innerHTML = 'Reservations';
+  buttonComments.id = `${index}`;
```

Comment:
- [ ] according to the requirements on the top of the home page there should be a counter that show the number of items (home)

Reply:
@mikethreels The counter is there, the thing is that the font makes it difficult to read. We changed the font so it will be easier.